### ISSUE TITLE
MP3 format from listen again - Audacity problem

### ISSUE THREAD

**Issue by caprenter** [caprenter/bcb-website#162]
Audacity does not recognise some MP3 from Listen Again so they have to be converted to WAV

FYI, this is what I use in the command line

ffmpeg -i input.mp3 output.wav

**Comment by notshi** [comment]
The following examples are links to the pages and the audio links from those pages.

1. [International Women's Day - Mon 8 Mar 8-9am](https://podcast.canstream.co.uk/bcb/index.php?id=57482) ([Audio Link](https://podcast.canstream.co.uk/bcb/get.php?fla=bcb-2021-03-08-45277.mp3))
2. [International Women's Day - Mon 8 Mar 2-3pm](https://podcast.canstream.co.uk/bcb/index.php?id=57489) ([Audio Link](https://podcast.canstream.co.uk/bcb/get.php?fla=bcb-2021-03-08-64603.mp3))

**Comment by caprenter** [comment]
I can't recreate this. My Audacity opens them fine.

I'm on Lubuntu 18.04. 
Audacity 2.2.1 - (from the repos probably)
I'm fairly sure I have lame and ffmpeg installed to help with the import and export.

**Comment by notshi** [comment]
Could be a bug? I'm on:

Ubuntu 20.10
Audacity 2.4.2

In any case, raising it for posterity.

**Comment by caprenter** [comment]
LAME and FFMPEG instructions - I'm sure you are aware. I can't remember where I am with these. I think LAME is now bundled or something...so yeah could be buggy. I think there is 2.3 version of audacity..?.?.?
https://manual.audacityteam.org/man/installing_and_updating_audacity_on_linux.html

I'm guessing there is something weird here though because it's only some shows from listen again right?

Normally you can open them without conversion - just these two examples (and I guess others, but it's not consistent?)

**Comment by notshi** [comment]
Yeah, it looks like I have the latest versions installed so it could be a bleeding-edge bug. Or not?

And yes, normally I can open them without conversion but ever so randomly, I get an unhappy MP3 that needs converting first.
